UserDataDownloadOrchestratorTrigger/index.ts

```diff
@@ -0,0 +1,7 @@
+import { Context } from "@azure/functions";
+import * as df from "durable-functions";
+
+export default (context: Context, input: unknown) =>
+  df
```

Comment:
what about moving the code which is currently into  `UserDataDownloadOrchestrator` here and
call `startNew` with  `UserDataDownload(Sub)Orchestrator` ? To avoid a layer of indirection

Reply:
It's actually a good idea, and it will help us when we'll have to handle DELETE requests too.